acceptance/tests/resource/service/windows.rb

```diff
@@ -37,23 +37,12 @@ def service_manifest(name, params)
   # 10 seconds, having the service sleep for 20
   # will supply the failure scenario on a failed
   # startup
-  mock_service_startfail = {
-    :name => "mock_service_startfail",
-    :start_sleep => 60,
+  mock_service_long_start_stop = {
```

Comment:
Since the timeout gives puppet the ability to start the service with a long startup, we no longer need two different mock services for long startup an long shutdown, we can just use the one and when we need it running we can use puppet with :timeout to start it.